### PR TITLE
Hooks documentation

### PR DESCRIPTION
**Pull request by vhf**

### CODE REVIEW

File: DICTIONARY.md [hunk 1]
```diff
@@ -14,14 +14,14 @@ in a `Livingdoc` and how users can interact with these `Components` in the
 
 See *Upstream*.
 
-## HuGO
+## [HuGO](http://www.sternwald.com/hugo/)
 
-A stand-alone product to manage images and agency reports in large numbers.
+A Digital Assets Management product edited by Sterwald, used to manage images and agency reports in large numbers.
```
Comment (gabrielhase) [inline DICTIONARY.md]: `edited` -> `developed and maintained`

File: SUMMARY.md [hunk 2]
```diff
@@ -54,6 +54,7 @@
 * [Metadata editor configuration](reference-docs/editor-configuration/metadata.md)
 * [Print API](reference-docs/server-print-api/print-api.md)
 * [Server config](reference-docs/server-configuration/config.md#services)
+  * [stack](reference-docs/server-configuration/hooks.md)
```
Comment (gabrielhase) [inline SUMMARY.md]: `stack` seems a strange name for this. Why not `hooks`?

File: reference-docs/server-extensions/hooks.md [hunk 3]
```diff
@@ -1,18 +1,27 @@
-# Livingdocs Server Publish Hooks
+# Livingdocs Server Hooks
 
 ## Overview
 
-The boilerplate app allows you to hook into the publish and unpublish transactions of articles and pages and implement your customizations, e.g., for third-party system integrations. Note that you are executing your code within the transaction thus don't do things like long-running polls in there. You can also abort the publish (visible to the user) by returning the callback with an error.
+The boilerplate app allows you to hook before publish and into the publish and unpublish transactions of articles and pages and implement your customizations, e.g., for third-party system integrations. Note that in the case of publish and unpublish hooks you are executing your code within the transaction, therefore it is best to avoid things like long-running polls in there. You can also abort the publish (visible to the user) by returning the callback with an error.
```
Comment (gabrielhase) [inline reference-docs/server-extensions/hooks.md]: -> `to hook into the 'before publish' state as well as the publish and unpublish transactions'

File: SUMMARY.md (hunk 2 above)
Comment (vhf) [reply SUMMARY.md]: Bad copy paste, good catch!

**Comment by vhf** [comment]
> From what I understood the only really new thing are the server-wide hooks. Correct?

Nope, `prepublishHooks` are also new. :)

**Pull request merged**